packages/react-components/react-carousel-preview/library/src/components/Carousel/Carousel.types.ts

```diff
@@ -43,7 +43,7 @@ export type CarouselProps = ComponentProps<CarouselSlots> & {
    * Enables mouse/touch drag on carousel items.
    * Defaults to: False
    */
-  enableDrag?: boolean;
+  draggable?: boolean;
```

Comment:
This should probably get a mention in the changefile

Reply:
Gonna move this one out to generic pre-stable updates, while we wait for above dependency issues to be merged.